code/code/spec/spec_objs_thief_quest.cc

```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
+
+#include "being.h"
+#include "comm.h"
+#include "extern.h"
+#include "obj_base_weapon.h"
+
+int thiefQuestWeapon(TBeing *victim, cmdTypeT command, const char *arg, TObj *object, TObj *) {
+  if (command != CMD_STAB && command != CMD_BACKSTAB && command != CMD_SLIT) return false;
+  if (strcmp(arg, "-special-") && command != CMD_STAB) return false;
+  if (!victim || !object) return false;
+
+  auto limb = command == CMD_BACKSTAB ? WEAR_BACK
+              : command == CMD_SLIT   ? WEAR_NECK
+                                      : limbStringToEnum(arg);
+
+  if (limb == WEAR_NOWHERE || limb == HOLD_RIGHT || limb == HOLD_LEFT || limb == MAX_WEAR)
+    return false;
+
+  auto limbDescription = victim->describeBodySlot(limb);
+
+  // 50% chance to proc on successful special attack
+  if (!::number(0, 1)) return false;
+
+  auto weapon = dynamic_cast<TBaseWeapon *>(object);
+  auto thief = weapon ? dynamic_cast<TBeing *>(weapon->equippedBy) : nullptr;
+  if (!weapon || !thief) return false;
+
+  auto damage = 0;
+  auto damageType = SKILL_STABBING;
+
+  if (command == CMD_STAB) {
+    damage = thief->GetMaxLevel() * 1.5;
```

Comment:
So an extra 150 damage on stab? up to 300 damage on slit

You have this going off 50% of the time so that could be a lot of damage from this proc. particularly for stab across a fight.

How much have you tested the impact of the damage here? My intuition is that it's a lot of added damage.

Would you consider toning the damage down? Maybe proc 33% and reduce the damage to 75 for stab and maybe up to 200 for slit.

Is it more powerful if you reduce damage and add some life drain or healing component when it procs?

My experience is people will be mad if we decide it's too good later and we want to nerf it :)

Reply:
No, you're right, it looks super strong. Vasco intentionally wanted it pretty strong, since he's planning to use it on a lvl 45-50 thief quest weapon, but he's also going to do playtesting of it himself and we'll tweak the power of it from there. Currently it's only used on one weapon in the game that no one even uses, and Vasco could just remove the proc from it temporarily if he needed to.

Also, unless I'm doing my math really wrong here, I intended the damage to be 75 on stab, 125 on backstab, and slit to be 150. Where are you getting the 150 for stab/300 for slit?

I'm up for any changes you guys think make sense - the original proc was super low chance and fairly low damage, so we just ramped it up quite a bit.

I tested it out quite a bit myself and it's definitely strong, but there's not really a good way to let Vasco test it otherwise, at this point. I really need to set up a dev server I can let others connect to.

Reply:
My brain isn't working. I think I was thinking skill level or something.